### DISCUSSION TITLE
Regression

### DISCUSSION THREAD

**Discussion by Gyuknj** [pmlmodelling/nctoolkit#50] [Q&A]
I have two datasets, namely; Sea surface temperature and 10m wind that span 1959 to 2022.
I used the script below written by you to compute the spatial correlation and it gave me good results.

import nctoolkit as nc
files = ["data/CMIP6_UKESM1-0-LL_Lmon_piControl_r1i1p1f2_gpp_1960-3059.nc",
"data/CMIP6_UKESM1-0-LL_Amon_piControl_r1i1p1f2_tas_1960-3059.nc"]
ds = nc.open_data(files)
ds.merge()
ds.cor_time(var1 = "gpp", var2 = "tas")
ds.plot()

I have been trying to compute also the spatial regression for the two datasets but no success.
Kindly, assist me with a similar script for the spatial regression if it is possible.

Thank you once again for your time and patience.

**Comment by robertjwilson** [comment]
As I said in a previous discussion, nctoolkit is not a package for statistical analysis. It does not offer the ability to do regressions. If there is missing functionality you are interested in adding, please raise an issue.
